src/account-manager.js

```diff
@@ -199,23 +199,30 @@ class AccountManager {
         // TODO: Remove unreturned addresses and add new returned addresses
     }
 
-    async export(accountId) {
+    async exportFile(accountId) {
         await this._launched;
-        const result = await this._invoke('export', null, {
+        const result = await this._invoke('exportFile', null, {
```

Comment:
`exportFile` and `exportWords` do not exist on the AccountsClient. The `method` is stil `export` but you can send flags of which type you want: https://github.com/nimiq/accounts/commit/4d29435e5e4490401a3c86a90788db0114b4ebfa#diff-20abd8c6e7eeac80d4ea800026b6cccaR87